spec/fixy/record_spec.rb

```diff
@@ -250,7 +250,6 @@ class PersonRecordWithErrorB < Fixy::Record
   let(:multibyte_record) { 'älimuk   Karil     ' }
   context 'with a record of multi-byte characters' do
     it 'should not raise with the right number of bytes' do
-      pending "This test does not work on Linux"
```

Comment:
this is not required anymore as spec is passing and [CI doesnt like it being pended](https://buildkite.com/gusto/fixy/builds/13#01884998-6d68-47b0-ad6e-1c5721781ec5)